cmd/webhook/main.go

```diff
@@ -59,8 +57,7 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	brokerv1beta1.SchemeGroupVersion.WithKind("Broker"): &brokerv1beta1.Broker{},
 
 	// For group messaging.cloud.google.com.
-	messagingv1alpha1.SchemeGroupVersion.WithKind("Channel"): &messagingv1alpha1.Channel{},
-	messagingv1beta1.SchemeGroupVersion.WithKind("Channel"):  &messagingv1beta1.Channel{},
```

Comment:
Why do we remove v1beta1 here? I'm wondering now maybe the default and validation webhook won't work for v1beta1 channel since we remove v1beta1 channel here.

Reply:
sorry I think the diff is a bit misleading, v1beta1 is not removed.

Reply:
> I believe it is also used in validation and defaults webhook. You need to search the usage of var types.
I think you are referring to this